api/restaurant1.json

```diff
@@ -0,0 +1,11 @@
+
+{
+  "id": "Vermont-Tap-House",
+  "name": "Vermont-Tap-House",
```

Comment:
Your names in your JSON can just be strings and don't need the dashes or anything. Will look nicer in your NavBar 👍